plugins/woocommerce-blocks/assets/js/base/components/state-input/state-input.tsx

```diff
@@ -104,7 +104,6 @@ const StateInput = ( {
 				errorMessage={ __( 'Please select a state.', 'woocommerce' ) }
 				errorId={ errorId }
 				required={ required }
-				autoComplete={ autoComplete }
```

Comment:
I removed this, these autoComplete's are auto complete hints like "country". I confirmed they were ignored by the old combobox and will be ignored by this new one. 

I've added a deprecated warning to instruct people not to pass autocomplete hints, but rather to choose from the values supported by Ariakit.

Reply:
This is no longer true, I have added support for autofill hints 🎉